### ISSUE TITLE
Egg Preservation

### ISSUE THREAD

**Issue by JeveGreen** [CleverRaven/Cataclysm-DDA#77597]
### Is your feature request related to a problem? Please describe.

It's not the first time I've brought up a Townsends video, and it'll probably not be the last, but [this video](https://www.youtube.com/watch?v=yUYgguMz1qI) actually got me thinking a bit: We don't have a lot of ways to preserve eggs in Cataclysm. At least not compared to most other food products.

We can either take a bunch and turn them into powder with a smoking rack/food dehydrator, or boil them and turn them into pickled eggs, both of which not only require a lot of eggs, all of the same kind, but also requires a lot of prep to do.

### Solution you would like.

So why not take a note out of [Townsends' video](https://www.youtube.com/watch?v=yUYgguMz1qI) and use a bit of wood ash or slaked lime to extend the shelf life of eggs? Wood ash is pretty much a waste product that's only useful for soap or clay items, available as soon as you start making fires if not earlier. And while slaked lime is harder to come by, it's a more effective preservant than ash, according to Townsends' source, and would add an alternative to turning your eggs into powder. But most importantly in both cases, this could be done with individual eggs (or the equivalent amount of smaller eggs,) something which we currently can't preserve outside of refridgeration.

### Additional context

I basically just copy-pasted my reddit-post on the subject, since I'm too lazy to write this all up again. Pardon my insolence.

**Comment by ampersand55** [comment]
Do we need to account for the build up of bacteria such as salmonella? That's would probably be the main reason for not including these recipes. 

Most official sources discourages the use of all non-refrigerated methods for the preservation of raw eggs due to the risk of salmonella, but only about [1 in 20000 eggs](https://pubmed.ncbi.nlm.nih.gov/11028959/) in the US carries Salmonella enteritidis. The risk of consuming at least one egg carrying salmonella after eating 100 eggs is about 1−0.99995^100 = 0.4988%. 

I personally think the risk is negligible.

> One method that has become very popular is storing eggs in lime water, also called “water glassing.” This method is not considered safe, nor is it a recommended method to store eggs for the long term.

> Water Glassing Is Dangerous

> Water glassing eggs was popular in the early 1900s. It was thought that eggs were sterile inside the hen and that all bacteria came through the shell after the eggs were laid. Research shows that Salmonella enteritidis can contaminate eggs as they are formed inside the hen’s reproductive organs, and there is no way a person can know if it is there.

https://extension.usu.edu/preserve-the-harvest/research/storing-eggs-safely

Note that salmonella bacteria can be killed by heating up all parts of the egg to 74 C (165 F), such as by hard boiling the egg.

**Comment by kevingranade** [comment]
I agree the risk is low enough we could neglect it initially, though a 1/20,000 salmonella chance would also be ok.

**Comment by JeveGreen** [comment]
> Do we need to account for the build up of bacteria such as salmonella? That's would probably be the main reason for not including these recipes.
> 
> Most official sources discourages the use of all non-refrigerated methods for the preservation of raw eggs due to the risk of salmonella, but only about [1 in 20000 eggs](https://pubmed.ncbi.nlm.nih.gov/11028959/) in the US carries Salmonella enteritidis. The risk of consuming at least one egg carrying salmonella after eating 100 eggs is about 1−0.99995^100 = 0.4988%.
> 
> I personally think the risk is negligible.
> 
> Note that salmonella bacteria can be killed by heating up all parts of the egg to 74 C (165 F), such as by hard boiling the egg.

Well we are preserving raw eggs, so that's kind of a non-issue, unless you like eating raw eggs regularly. They're meant to serve as an ingredient, not as a standalone meal, so one could simply add the "raw" tag to preserved eggs.

**Comment by ampersand55** [comment]
It was a potential issue as fresh raw eggs with traces of salmonella and raw egg with 8 months of salmonella growth have different amounts of the bacteria. You often just cook eggs until they coagulate, which occurs between 60-70 C, which is not enough to kill all the bacteria.

Another thing that bears mentioning is that store-bought eggs in the US are washed and sanitized, which removes the protective bloom and makes them unsuitable for non-refrigeration methods of preserving. 

https://www.farmhouseonboone.com/water-glassing-eggs/
